src/locales/en/translation.json

```diff
@@ -135,6 +135,7 @@
     "reclaimedAmount": "Amount to reclaim"
   },
   "errors": {
+    "LedgerDerivedDifferentAccount": "This Ledger did not derive this account.",
```

Comment:
In terms of the copy, would something like this be even clearer and easier to understand?

_This account does not belong to the currently connected Ledger._